dartvlc/main.hpp

```diff
@@ -20,4 +20,4 @@
 
 /* libVLC++ wrapper for dart_vlc */
 
```

Comment:
I believe changing extension of the file may cause problem on Linux (since this was compiled directly with `add_library`). Will need a bit of rework.